extensions/amp-ad/0.1/test/test-concurrent-load.js

```diff
@@ -75,13 +75,27 @@ describes.realWin('concurrent-load', {}, env => {
       installTimerService(win);
     });
 
-    it('should throttle ad loading one per second', () => {
+    it('should throttle ad loading one per second', function* () {
       expect(is3pThrottled(win)).to.be.false;
       incrementLoadingAds(win);
       expect(is3pThrottled(win)).to.be.true;
       clock.tick(999);
+      yield;
```

Comment:
This is genius, but I don't fully understand how it works. Or this is a bit awkward.

Wouldn't it be better to always yield a promise instead of relying on the magic setInterval shortcircuiting in the trivial case?

If you return a promise from `incrementLoadingAds` that would work for this test.